### ISSUE TITLE
Support for LTX-Video diffusers pipeline

### ISSUE THREAD

**Issue by nitinmukesh** [junhahyung/STGuidance#13]
Please build the support for LTX-Video diffusers pipeline
https://github.com/huggingface/diffusers/tree/main/src/diffusers/pipelines/ltx

**Comment by kinam0252** [comment]
Hi, thank you for the great suggestion! We'll work on building support for LTX Video pipeline :)

**Comment by nitinmukesh** [comment]
Thank you @kinam0252 .

Looks like you will be busy in the next year as more and more such project are being integrated in Diffussers and your code (STGuidance) is awesome. :)

**Comment by kinam0252** [comment]
Thank you for your kind words! 😊

**Comment by kinam0252** [comment]
Thank you for waiting so long. I have implemented STG in the LTX-Video Diffusers pipeline and uploaded it [here](https://github.com/junhahyung/STGuidance/tree/main/diffusers/ltxvideo). Please try it out, and let me know if you encounter any issues!

**Comment by nitinmukesh** [comment]
TQVM @kinam0252 .

I will try it now. I guess I will have to build diffusers from source?

**Comment by kinam0252** [comment]
Yes, ```pip install diffusers``` and run ```python test.py``` should work.

**Comment by nitinmukesh** [comment]
Hello @kinam0252 

Please also update for LTXImageToVideoPipeline pipeline. Thank you

**Comment by kinam0252** [comment]
Sure, just uploaded LTX-I2V pipeline [here](https://github.com/junhahyung/STGuidance/tree/main/diffusers/ltxvideoI2V)!

**Comment by nitinmukesh** [comment]
TQVM. You are awesome.
